src/ripple/protocol/AnyPublicKey.h

```diff
@@ -102,9 +102,16 @@ class AnyPublicKey
 
 public:
     AnyPublicKey() = delete;
-    AnyPublicKey (AnyPublicKey const&) = delete;
     AnyPublicKey& operator= (AnyPublicKey const&) = delete;
 
+    AnyPublicKey (AnyPublicKey const& other)
+        : buffer_type(other.buffer_type::member.data(),
+            other.buffer_type::member.size())
+        , AnyPublicKeySlice (buffer_type::member.data(),
+            buffer_type::member.size())
+    {
+    }
```

Comment:
The existence of a copy constructor implies we are making needless copies somewhere. If this copy constructor is commented out, which line then breaks?